.gitignore

```diff
@@ -21,3 +21,4 @@ __pycache__
 *.swo
 *~
 htmlcov
+python-env
```

Comment:
Not resolved.
<img width="366" alt="Screenshot 2022-11-02 at 21 48 59" src="https://user-images.githubusercontent.com/3709715/199600015-2bd19d6c-ce83-4560-bfe6-64b433343027.png">